src/components/tutorial-meta/components/interactive-lab-button/index.tsx

```diff
@@ -1,8 +1,10 @@
 import { IconTerminalScreen16 } from '@hashicorp/flight-icons/svg-react/terminal-screen-16'
+import Button from 'components/button'
 import { useInstruqtEmbed } from 'contexts/instruqt-lab'
-import s from './interactive-lab-button.module.css'
 
-// @TODO replace with Button component once implemented
+/**@TODO this button shows a drop-shadow elevation in the design,
```

Comment:
[See this ticket](https://app.asana.com/0/1202001387788447/1202073130822761)

Reply:
Just checked [the DS Button component](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/?node-id=2%3A9) for this, elevation is definitely missing on the primary style buttons! Also noticed that the elevation should go away in the `focus-visible` state, so those are listed here too.

I believe the updates that need to be made are:

- Add the `box-shadow` property to the `.primary` class:
  ```css
  box-shadow: var(--token-elevation-low-box-shadow);
  ```
- Set `box-shadow: none` in `.primary:focus-visible` state
- Set `box-shadow: none` in `.secondary:focus-visible` state
- Set `box-shadow: none` in `.critical:focus-visible` state

Reply:
FYI elevation style in HDS Button is not applied for `active` and `focused` states: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components-%5BWIP%5D?node-id=2340%3A21001

Reply:
Thank you @didoo! We've got the active states covered ([see here](https://github.com/hashicorp/dev-portal/blob/a13b86ac4998ed58b1533c4891f21629834b15c3/src/components/button/button.module.css)), the focus ones are what we haven't handled yet. Just to check, the `disabled` states don't have elevation applied either right?

Reply:
Yes, `disabled` also doesn't have elevation (essentially elevation is only for `default` + `hover` :)